Description: update base template for Sphinx 1.3
 The 'default' theme has been renamed to 'classic' in Sphinx 1.3.
Author: Dmitry Shachnev <mitya57@debian.org>
Forwarded: https://github.com/morse-simulator/morse/pull/652
Last-Update: 2015-08-24

--- a/doc/morse-theme/templates/layout.html
+++ b/doc/morse-theme/templates/layout.html
@@ -1,7 +1,7 @@
 {#
  MORSE Sphinx template
 #}
-{% extends "default/layout.html" %}
+{% extends "classic/layout.html" %}
 
 {% block rootrellink %}
     <li><a href="{{ pathto(master_doc) }}">{{ project }}</a>{{ reldelim1 }}
